project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,217 @@
+   `terraform-aws-example-microservice-infra`
+   `terraform-aws-example-network`
+   ![img_9.png](../readme-images/create_tfc_workspace_ss.png)
+
+
+2. Go to Settings for the created TFC workspaces, Under `Remote State Sharing` allow `Share with all workspaces in this organization`.
+   ![img_10.png](../readme-images/tfc_remote_sharing_ss.png)
+
+
+3. Create a [global variable set](https://developer.hashicorp.com/terraform/tutorials/cloud-get-started/cloud-workspace-create#create-the-workspace) for your AWS Credentials.
+
+### AWS Steps with Terraform
+
+Create the baseline infrastructure with the following steps in your CLI.
+
+1. Navigate to the `terraform-aws-example-network` directory found in this example and run `terraform init` and the `terraform apply` command.
+
+2. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example and run `terraform init` and the  `terraform apply` command.
+
+
+### Terraform Module Creation
+
+TODO(Teresa): Make this repo public.
+
+1. Fork the `hashicorp/terraform-aws-example-microservice-ecs-all-envs` repository in Github [here](https://github.com/hashicorp/terraform-aws-example-microservice-ecs-allenvs) to your own Github organization.
+
+2. Run the following git commands in the forked repository's working directory.
+
+```shell
+go tag v.0.0.1
+go push --tags
+```
+
+#### Publish The Example  Module
+1. Inside Terraform Cloud, Click Registry and Select `Publish a Module`
+![img.png](../readme-images/tfc_publish_module_ss.png)
+
+
+2. Select Github under `Version Control Provider`
+
+![img_1.png](../readme-images/publish_module_vcs_ss.png)
+
+3. Select your forked repository of the no-code module created in the last step.
+
+![img_2.png](../readme-images/select_repo_ss.png)
+
+4. Click `Publish Module`
+
+![img_3.png](../readme-images/publish_module.png)
+
+5. Review the published module details
+
+![img_4.png](../readme-images/review_module_details.png)
+
+
+
+![img_8.png](../readme-images/tfc_registry_ss.png)
+
+
+## Waypoint Template Steps 
+
+TODO(Teresa): Replace figma images with proper screenshots once UI is ready.
+
+
+1. Set up your TFC Credentials in Waypoint. You will need a TFC team access token found in Terraform Cloud and your TFC Organization Name.
+![img.png](../readme-images/waypoint_tfc_creds_setup_ss.png)
+
+
+2. Create a Waypoint Project Template by filling in the fields in the Template Creation page:
+
+Your project template will be utilized to generate the waypoint hcl for developers to start off their projects with. This template will contain
+Terraform details that will be used to spin up resources that will be used for application deployment. Please see the following example project
+template: 
+
+```shell
+// this is the simple example for now
```

Comment:
I think this line can be removed now. IIRC we added this here to indicate that this configuration was derived from deploying to the single-env example for the `waypoint-ecs` Terraform module, but it has since changed.